lld/ELF/Arch/Cheri.h

```diff
@@ -60,12 +60,14 @@ struct CheriCapRelocLocation {
 };
 
 struct CheriCapReloc {
+  bool isCode;
```

Comment:
A comment here or on addCapReloc() might be nice, just to make future readers don't think they need to set isCode=true for functions.